rust/config/mainnet/arbitrum_config.json

```diff
@@ -48,7 +48,7 @@
       "domain": "1667591279",
```

Comment:
I'm also realizing we need to change the `reorgPeriod` env vars that are used by our validators for Celo/Alfajores/Avalanche/Fuji:
Testnet2:
https://github.com/abacus-network/abacus-monorepo/blob/main/typescript/infra/config/environments/testnet2/agent.ts#L28
Mainnet:
https://github.com/abacus-network/abacus-monorepo/blob/main/typescript/infra/config/environments/mainnet/agent.ts#L28
(I'm very aware how ridic it is to need to change values in so many places 😅 but until we do https://github.com/abacus-network/abacus-monorepo/issues/532, the configuration is separate)